src/workloads/scale/LLTInsert.yml

```diff
@@ -167,13 +191,12 @@ Actors:
         GenerateCollectionNames: true
         CollectionSortOrder: forward
         FindOptions:
-          BatchSzie: 1000
-          Hint: _id_ # Currently only support the index name.
+          BatchSize: 1000
+          Hint: _id_
           Comment: SnapshotScannerShort
 
-  # <-----  4 threads @ 25% in batches of 50
 - Name: InsertBenchmark.Low.50
-  Type: MultiCollectionInsert
```

Comment:
Ooh so seems like maybe MultiCollectionInsert can be axed?

Reply:
yes, I will do that now! Less is more. (TBH maybe all the MCX could be removed ).